README.md

```diff
@@ -1,77 +1,141 @@
+### `mssql_accept_microsoft_cli_utilities_for_sql_server_eula`
 
-This section is optional.  Some roles may export variables for playbooks to
-use later.  These are analogous to "return values" in Ansible modules.  For
-example, if a role performs some action that will require a system reboot, but
-the user wants to defer the reboot, the role might set a variable like
-`template_reboot_needed: true` that the playbook can use to reboot at a more
-convenient time.
+Set this variable to `true` to indicate that you accept EULA for installing the
+`mssql-tools` package.
 
-Example:
+The license terms for this product can be downloaded
+from <http://go.microsoft.com/fwlink/?LinkId=746949> and found in
+`/usr/share/doc/mssql-tools/LICENSE.txt`.
 
-`template_reboot_needed` - default `false` - if `true`, this means
-a reboot is needed to apply the changes made by the role
+Default: `false`
 
-## Dependencies
+### `mssql_accept_microsoft_sql_server_2019_standard_eula`
 
-A list of other roles hosted on Galaxy should go here, plus any details in
-regards to parameters that may need to be set for other roles, or variables
-that are used from other roles.
+Set this variable to `true` to indicate that you accept EULA for using the
+`mssql-conf` utility.
 
-## Example Playbook
+The license terms for this product can be found in `/usr/share/doc/mssql-server`
+or downloaded from <https://go.microsoft.com/fwlink/?LinkId=2104078&clcid=0x409>.
+The privacy statement can be viewed at
+<https://go.microsoft.com/fwlink/?LinkId=853010&clcid=0x409>.
+
+Default: `false`
+
+### `mssql_password`
+
+The password for the database sa user. The password must have a minimum length
+of 8 characters, include uppercase and lowercase letters, base 10 digits or
+non-alphanumeric symbols. Do not use single quotes ('), double quotes ("), and
+spaces in the password because `sqlcmd` cannot authorize when the password
+includes those symbols.
+
+When running this role on a host that has MSSQL set up, the mssql_password
+variable overwrites the existing sa user password to the one that you specified.
+
+Default: `None`
+
+### `mssql_edition`
+
+The edition of MSSQL to install. Use one of the following values:
+
+* `Enterprise`
+* `Standard`
+* `Web`
+* `Developer`
+* `Express`
+* `Evaluation`
+* A product key in the form `#####-#####-#####-#####-#####`, where `#` is a
+  number or a letter.
+  For more information, see
+  <https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver15>.
+
+Default: `None`
+
+### `mssql_tcp_port`
+
+The port that MSSQL listens on.
 
-Including an example of how to use your role (for instance, with variables
-passed in as parameters) is always nice for users too:
+Default: `1443`
+
+### `mssql_ip_address`
+
+The IP address that MSSQL listens on.
+
+WARNING: The role applies the same IP address setting to all hosts that you run
+the role against. You cannot specify a different IP address for a different
```

Comment:
Why not? Set the variable to some Jinja2 template with variable expansion (using. e.g. some Ansible fact). Am I missing something?

Reply:
Maybe we should just get rid of this warning.

Reply:
Sure, I did not think about it this way at all. Thanks, Pavel, I have removed the warning.